### PR TITLE
Fix margin speaker_tag_line

### PR DESCRIPTION
**Pull request by tebasakyu**
## Issue
- None.

## Overview (Required)
- Speaker tag line text margins(start & end) are not working.
- I use padding instead of margin. It's working.

## Screenshot
Before | After
:--: | :--:
 |

### CODE REVIEW

**Comment by takahirom** [comment]
👀

File: app/src/main/res/layout/fragment_speaker_detail.xml [hunk 1]
```diff
@@ -57,10 +57,10 @@
             android:id="@+id/speaker_tag_line"
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
-            android:layout_marginEnd="8dp"
-            android:layout_marginStart="8dp"
             android:layout_marginTop="8dp"
             android:paddingBottom="32dp"
+            android:paddingStart="8dp"
+            android:paddingEnd="8dp"
```
Comment (takahirom) [inline app/src/main/res/layout/fragment_speaker_detail.xml]: Probably we can fix this by using layout_width="0dp". We can layout it by Constraint. 👍 What do you think?

```
   android:layout_width="0dp"
```

**Comment by takahirom** [comment]
Thank you for catching this problem 👍

File: app/src/main/res/layout/fragment_speaker_detail.xml (hunk 1 above)
Comment (tebasakyu) [reply app/src/main/res/layout/fragment_speaker_detail.xml]: Thank you for review and comment 😃 
I tried your solution, but please look Screenshot. 
I think you expected  text left alignment. It's OK?

Comment (takahirom) [reply app/src/main/res/layout/fragment_speaker_detail.xml]: Thank you for checking! I think before is better 😄

**Comment by takahirom** [comment]
LGTM 👍

**Pull request merged**